src/snowflake/cli/plugins/nativeapp/exceptions.py

```diff
@@ -23,16 +23,16 @@ def __init__(self, name: str):
 
 
 class ApplicationAlreadyExistsError(ClickException):
-    """An application not created by SnowCLI exists with the same name."""
+    """An application object not created by Snowflake CLI exists with the same name."""
 
     def __init__(self, name: str):
         super().__init__(
-            f'A non-dev application "{name}" already exists in the account.'
+            f'A non-dev application object "{name}" already exists in the account.'
```

Comment:
What does "non-dev" mean here? We have the "development mode" which allows providers to see the contents of the application object. Is that the case here?

Reply:
Yes, but specifically we are referring to "using files on a named stage"/loose files mode within the development mode. Do you have a suggestion on how we can improve this?

Reply:
Maybe "An application object created in development mode." Loose file mode is kind of an outage in the docs, currently. I have a JIRA to add more information about that.